JSDemos/Demos/Scheduler/CurrentTimeIndicator/description.md

```diff
@@ -1 +1,5 @@
-This example shows how to&nbsp;enable the current time indication. The switches under the Scheduler allow you to&nbsp;specify how to&nbsp;indicate the current time: by&nbsp;using a&nbsp;specific indicator, shading or&nbsp;both of&nbsp;them. You can also specify how often the Scheduler updates their position using the *&laquo;Update position in&raquo;* number box under the component.
\ No newline at end of file
+The Scheduler can indicate current time if you enable the [showCurrentTimeIndicator](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#showCurrentTimeIndicator) property. Use the [indicatorUpdateInterval](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#indicatorUpdateInterval) property to specify how frequently the indicator should change its position.
```

Comment:
Minor thing - I would just say "indicates current time if you enable". Otherwise the phrase is a bit vague... So I enable, and then the control "can" indicate? So maybe it will or maybe it won't? 

```suggestion
The Scheduler indicates current time if you enable the [showCurrentTimeIndicator](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#showCurrentTimeIndicator) property. Use the [indicatorUpdateInterval](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#indicatorUpdateInterval) property to specify how frequently the indicator should change its position.
```